### PR TITLE
Adapt PyExperimenter to use its own logger

### PR DESCRIPTION
**Pull request by LukasFehring**
I adapted the PyExperimenter to use its own logger as explained in the corresponding issue.

### CODE REVIEW

File: py_experimenter/database_connector_mysql.py [hunk 1]
```diff
@@ -13,13 +13,13 @@
 class DatabaseConnectorMYSQL(DatabaseConnector):
     _prepared_statement_placeholder = '%s'
 
-    def __init__(self, experiment_configuration: ConfigParser, use_codecarbon:bool, codecarbon_config:ConfigParser, database_credential_file_path:str):
+    def __init__(self, experiment_configuration: ConfigParser, use_codecarbon:bool, codecarbon_config:ConfigParser, database_credential_file_path:str, logger_name:str = 'py_experimenter'):
```
Comment (tornede) [inline py_experimenter/database_connector_mysql.py:16]: Default: `PyExperimenter`

Actually I am wondering if we really need the default here? Isn't it enough to have it in the constructor of `PyExperimenter`

File: py_experimenter/experimenter.py [hunk 2]
```diff
@@ -28,10 +28,15 @@ def __init__(self,
                  table_name: str = None,
                  database_name: str = None,
                  use_codecarbon: bool = True,
-                 name='PyExperimenter'):
-        """
-        Initializes the PyExperimenter with the given information.
-
+                 name='PyExperimenter',
+                 logger_name:str = 'py_experimenter',
```
Comment (tornede) [inline py_experimenter/experimenter.py:32]: Default: `PyExperimenter`

**Review by tornede — CHANGES_REQUESTED** [review]
LGTM and works fine, with one exception about the default logger name (see comments).

File: py_experimenter/database_connector_mysql.py (hunk 1 above)
Comment (LukasFehring) [reply py_experimenter/database_connector_mysql.py]: You're correct, i fixed that

**Review by tornede — APPROVED** [review]
LGTM

**Pull request merged**